libs/auth/src/card_reader.ts

```diff
@@ -86,10 +86,6 @@ export class CardReader {
     });
   }
 
-  getReaderStatus(): Reader['status'] {
-    return this.reader.status;
-  }
```

Comment:
Realized that this method wasn't getting used anywhere so I removed it (the `onReaderStatusChange` listener covers all needs for now)

Reply:
Nice, I was wondering earlier if both were necessary